### ISSUE TITLE
Laravel 6/7 install problems

### ISSUE THREAD

**Issue by azcoppen** [voryx/Thruway#342]
Symfony event dispatcher 4 doesn't play well with Laravel - needs 5+

For anyone running into this, until a new version is released this will work:

```
        "symfony/event-dispatcher": "5.0.8 as 4.4.8",
        "thruway/pawl-transport": "^0.5.1",
        "voryx/thruway": "dev-master",
```

**Comment by xbrunosousa** [comment]
Not work in Laravel 7 (PHP 7.4) :(
![image](https://user-images.githubusercontent.com/18221685/83962876-a3bf0180-a877-11ea-877c-1f2ece9ff3c2.png)

**Comment by azcoppen** [comment]
@xbrunosousa Symfony 5 completely changed the event schema - there's a PR for it but the repo maintainers seem to have dismissed it.

**Comment by mbonneau** [comment]
@xbrunosousa I would still like to see the EventDispatcher completely decoupled from Symfony as I said here: https://github.com/voryx/Thruway/pull/331#issuecomment-560001092

**Comment by azcoppen** [comment]
@mbonneau That's great, but when it is going to happen? That comment is 6+ months old. Not wanting to be combative here, just saying: how long is this (excellent) package going to be Symfony 4-only?

**Comment by davidwdan** [comment]
Closed by #343
